package.json

```diff
@@ -27,13 +27,16 @@
     "start": "react-scripts-ts start",
     "build": "react-scripts-ts build",
     "test": "react-scripts-ts test --env=jsdom",
-    "eject": "react-scripts-ts eject"
+    "eject": "react-scripts-ts eject",
+    "styleguide": "styleguidist server",
+    "styleguide:build": "styleguidist build"
   },
   "devDependencies": {
     "@types/jest": "22.1.3",
     "@types/node": "9.4.6",
     "@types/react": "16.0.38",
     "@types/react-dom": "16.0.4",
+    "react-styleguidist": "^6.2.6",
```

Comment:
Do we want the carret?

Reply:
As i'm pretty sure we want to use greenkeeper for the geostyler the carret should be removed. I take care of it.